scripts/buildImage.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+cd "$(dirname -- "$0")/../"
```

Comment:
As we don't want to add files to the image, we don't need to run within the project root.
So the docker context (i.e. cwd) can be the scripts directory.